rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +287,31 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
-                    tx = deserialize_ethereum_transaction(  # type: ignore
-                        data=entry,
-                        internal=is_internal,
-                        ethereum=None,
-                    )
+                    # Handle genesis block transactions
+                    if entry['hash'].startswith('GENESIS') is False:
+                        tx = deserialize_ethereum_transaction(  # type: ignore
+                            data=entry,
+                            internal=is_internal,
+                            ethereum=None,
+                        )
+                    else:
+                        # Handling genesis transactions
+                        dbtx = DBEthTx(self.db)  # type: ignore
+                        tx = dbtx.get_or_create_genesis_transaction(account=account)
+                        trace_id = dbtx.get_max_trace_id(GENESIS_HASH)
+                        trace_id = trace_id + 1 if trace_id is not None else 0
```

Comment:
Can you perhaps add this to get_max_trace_id() and adjust its docstring? Since the function is only use in one place this would make sense.

Reply:
Done